Model/User.php

```diff
@@ -190,9 +176,7 @@ public function unserialize($serialized)
             $this->salt,
             $this->usernameCanonical,
             $this->username,
-            $this->expired,
             $this->locked,
-            $this->credentialsExpired,
```

Comment:
@XWB It looks great. I can confirm that I just tested this locally: serializing using the `User` in the latest 1.3.5 tag, and then unserializing into the new `User` that's in this PR. Everything worked perfectly!

👍